### PR TITLE
Add sample app deployment to docs.

### LINKED ISSUE #3002: Add example application deployment to docs
## Description

@thesteve0 
Do we want to add an example in doc showing how to Deploy a Test Workload.
If so do we want to reuse one of the ones listed here or provide something new

Here is a simple example that @thesteve0 tested that works:
https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-5DFC347C-694B-4288-96DA-EAEB5818D951.html

Here is another example that needs a load balancer
https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-tanzu-k8s-clusters-tutorial.html

### PR DESCRIPTION
**Pull request by kcoriordan**
Signed-off-by: kcoriordan <koriordan@vmware.com>

## What this PR does / why we need it
Adds steps for deploying a sample workload to a workload cluster.

@thesteve0 - added this under Reference section for now while it's being reviewed/tested and decide then where it should go once it's verified.  

## Details for the Release Notes (PLEASE PROVIDE)

```release-note

```

## Which issue(s) this PR fixes

Fixes: #3002

### CODE REVIEW

**Comment by thesteve0** [comment]
LGTM

**Comment by joshrosso** [comment]
_retriggered tests_

@kcoriordan if we include this change, would it make sense to add to the bottom of `Next Steps` in Getting Started guides?

image

File: docs/site/content/docs/latest/sample.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+
+# Deploy a Test Workload to a Workload Cluster
```
Comment (joshrosso) [inline docs/site/content/docs/latest/sample.md:2]: This would also work for `unmanaged-cluster`s. Could we make this generic to include for both?

**Comment by kcoriordan** [comment]
Closing and Replacing this PR with https://github.com/vmware-tanzu/community-edition/pull/3666
as this was based on older folder structure using` /latest`

**Pull request closed without merging**